packages/ui-components/src/icon/labicon.tsx

```diff
@@ -695,6 +710,11 @@ export namespace LabIcon {
      * Optional title that will be set on the icon's outermost container node
      */
     title?: string;
+
+    /**
+     * Optional slot property to specify the position of the icon in the template
+     */
+    slot?: 'end' | 'start' | null;
```

Comment:
let's allow any string here as slot is not specified in details.

```suggestion
    slot?: string | null;
```